cns/api.go

```diff
@@ -37,6 +37,7 @@ type HTTPService interface {
 	SendNCSnapShotPeriodically(int, chan bool)
 	SetNodeOrchestrator(*SetOrchestratorTypeRequest)
 	SyncNodeStatus(string, string, string, json.RawMessage) (int, string)
+	SyncHostNCVersion(string)
```

Comment:
I dont think this API makes sense here. this interface should only exposed APIs which are exposed by service and be invoked by clients or other components (like CNI)

Reply:
I'll removed it from interface.